M/MPFR/build_tarballs.jl

```diff
@@ -3,12 +3,12 @@
 using BinaryBuilder
 
 name = "MPFR"
-version = v"4.1.0"
+version = v"4.0.2"
 
 # Collection of sources required to build MPFR
 sources = [
     "https://www.mpfr.org/mpfr-current/mpfr-$(version).tar.xz" =>
-    "0c98a3f1732ff6ca4ea690552079da9c597872d30e96ec28414ee23c95558a7f",
+    "1d3be708604eae0e42d578ba93b390c2a145f17743a744d8f3f8c2ad5855a38a",
```

Comment:
I guess I didn't even need this, as we are reusing the binaries?

Reply:
I think that strictly speaking you're correct